src/Compilers/CSharp/Portable/Lowering/AsyncRewriter/AsyncRewriter.AsyncIteratorRewriter.cs

```diff
@@ -34,6 +34,15 @@ private sealed class AsyncIteratorRewriter : AsyncRewriter
                 // PROTOTYPE(async-streams): Why does AsyncRewriter have logic to ignore accessibility?
             }
 
+            protected override void VerifyPresenceOfRequiredAPIs(DiagnosticBag bag)
+            {
+                EnsureWellKnownMember(WellKnownMember.System_Collections_Generic_IAsyncEnumerable_T__GetAsyncEnumerator, bag);
+                EnsureWellKnownMember(WellKnownMember.System_Collections_Generic_IAsyncEnumerator_T__WaitForNextAsync, bag);
+                EnsureWellKnownMember(WellKnownMember.System_Collections_Generic_IAsyncEnumerator_T__TryGetNext, bag);
+                EnsureWellKnownMember(WellKnownMember.System_IAsyncDisposable__DisposeAsync, bag);
+                EnsureWellKnownMember(WellKnownMember.System_Threading_Tasks_ValueTask_T__ctor, bag);
+            }
```

Comment:
>} [](start = 12, length = 1)

Do we need to call `base.VerifyPresenceOfRequiredAPIs()`?

Reply:
Good catch. FIxed and covered.